components/Admin/AdminButton.tsx

```diff
@@ -160,30 +301,50 @@ function AdminButton({
       // @ts-ignore
       overrideTargetButton={styles.overrideTargetButton}
       withDownIcon={false}
-      onClose={() => setIsOpen(false)}
+      onClose={
+        () => {
+          if (isExcludeSubmenuOpen || isFeaturedSubmenuOpen) {
+            return setIsOpen(true)
+          }          
+          return setIsOpen(false)
+      }}
```

Comment:
easier to read. Please run formatter
```suggestion
      onClose={
        () => ((isExcludeSubmenuOpen || isFeaturedSubmenuOpen) ? setIsOpen(true) : setIsOpen(false))
       }
```